xml/System.Windows.Forms/StatusBar.xml

```diff
@@ -1379,7 +1379,7 @@ This class is not available in .NET 5.0 and later versions. Use the <see cref="T
       <Parameters />
       <Docs>
         <summary>Returns a string representation for this control.</summary>
-        <returns>String</returns>
+        <returns>To be added.</returns>
```

Comment:
It does return something: https://referencesource.microsoft.com/#System.Windows.Forms/winforms/Managed/System/WinForms/StatusBar.cs,50566e4e57e7fc20

Better to still be compliant

```suggestion
        <returns>A string representation for this control.</returns>
```

Reply:
Thanks for catching these. I thought I'd double-checked all my diffs for places where I'd replaced with "To be added".